testsuite/MDAnalysisTests/topology/test_pqr.py

```diff
@@ -34,32 +30,28 @@
 
 
 class TestPQRParser(ParserBase):
-
-    __test__ = True
-
     parser = mda.topology.PQRParser.PQRParser
     filename = PQR
-    expected_attrs = ['ids', 'names', 'charges', 'radii',
-                      'resids', 'resnames', 'icodes',
-                      'segids']
+    expected_attrs = [
+        'ids', 'names', 'charges', 'radii', 'resids', 'resnames', 'icodes',
+        'segids'
+    ]
     guessed_attrs = ['masses', 'types']
     expected_n_atoms = 3341
     expected_n_residues = 214
     expected_n_segments = 1
 
     def test_attr_size(self, top):
```

Comment:
@jbarnoud would you be satisfied if these changes were implemented, i.e., can you upgrade your review from an informal comment to "changes requested" so that this can move along?